README.md

```diff
@@ -81,10 +113,9 @@ changes from feature branches into your `main` branch.
   - [CSS Box Model](http://www.w3schools.com/css/css_boxmodel.asp)
```

Comment:
I can't directly comment on it because it's not quite within the changes but directly above this line there is the line:
```md
  - [A guide to font families](http://www.w3schools.com/cssref/css_websafe_fonts.asp)
```

This particular guide is kinda out of date as it doesn't include the concept of using [system fonts](https://css-tricks.com/snippets/css/system-font-stack/) which is a fairly common practice these days. (This is not a critical or blocking issue by any means, feel free to ignore.)

Reply:
Looking over the page you linked to, that adds significant complications, and still seems pretty fluid, where the "websafe fonts" will still work and in a generally predictable way. So I think I'm going to skip this for now. Feel free to extract it out into an issue, though, especially if there's a "simple" way to deal with this that doesn't involve listing a pile of system specific fonts.